dbt/include/glue/macros/adapters.sql

```diff
@@ -1,96 +1,3 @@
-    using {{ file_format }}
-  {%- endif %}
-{%- endmacro -%}
-
-
-
-{% macro partition_cols(label, required=false) %}
-  {%- set cols = config.get('partition_by', validator=validation.any[list, basestring]) -%}
-  {%- if cols is not none %}
-    {%- if cols is string -%}
-      {%- set cols = [cols] -%}
-    {%- endif -%}
-    {{ label }} (
-    {%- for item in cols -%}
-      {{ item }}
-      {%- if not loop.last -%},{%- endif -%}
-    {%- endfor -%}
-    )
-  {%- endif %}
-{%- endmacro -%}
-
-{% macro comment_clause() %}
-  {%- set raw_persist_docs = config.get('persist_docs', {}) -%}
-
-  {%- if raw_persist_docs is mapping -%}
-    {%- set raw_relation = raw_persist_docs.get('relation', false) -%}
-      {%- if raw_relation -%}
-      comment '{{ model.description | replace("'", "\\'") }}'
-      {% endif %}
-  {%- else -%}
-    {{ exceptions.raise_compiler_error("Invalid value provided for 'persist_docs'. Expected dict but got value: " ~ raw_persist_docs) }}
-  {% endif %}
-{%- endmacro -%}
-
-{% macro clustered_cols(label, required=false) %}
-  {%- set cols = config.get('clustered_by', validator=validation.any[list, basestring]) -%}
-  {%- set buckets = config.get('buckets', validator=validation.any[int]) -%}
-  {%- if (cols is not none) and (buckets is not none) %}
-    {%- if cols is string -%}
-      {%- set cols = [cols] -%}
-    {%- endif -%}
-    {{ label }} (
-    {%- for item in cols -%}
-      {{ item }}
-      {%- if not loop.last -%},{%- endif -%}
-    {%- endfor -%}
-    ) into {{ buckets }} buckets
-  {%- endif %}
-{%- endmacro -%}
-
-{% macro location_clause() %}
-  {%- set location_root = config.get('location', validator=validation.any[basestring]) -%}
-  {%- set identifier = model['alias'] -%}
-  {%- if location_root is not none %}
-    location '{{ location }}/{{ identifier }}'
-  {%- endif %}
-{%- endmacro -%}
-
-{% macro options_clause() -%}
-  {%- set options = config.get('options') -%}
-  {%- if config.get('file_format') == 'hudi' -%}
-    {%- set unique_key = config.get('unique_key') -%}
-    {%- if unique_key is not none and options is none -%}
-      {%- set options = {'primaryKey': config.get('unique_key')} -%}
-    {%- elif unique_key is not none and options is not none and 'primaryKey' not in options -%}
-      {%- set _ = options.update({'primaryKey': config.get('unique_key')}) -%}
-    {%- elif options is not none and 'primaryKey' in options and options['primaryKey'] != unique_key -%}
-      {{ exceptions.raise_compiler_error("unique_key and options('primaryKey') should be the same column(s).") }}
-    {%- endif %}
-  {%- endif %}
-
-  {%- if options is not none %}
-    options (
-      {%- for option in options -%}
-      {{ option }} "{{ options[option] }}" {% if not loop.last %}, {% endif %}
-      {%- endfor %}
-    )
-  {%- endif %}
-{%- endmacro -%}
-
-{% macro glue__check_schema_exists(database, schema) -%}
-    {{ return(adapter.dispatch('schema_exists')(database, schema)) }}
-{% endmacro %}
```

Comment:
`adapter.dispatch` will look for the following macros (in this order):
1. `glue__`
2. `spark__`
3. `default__`

for that reason, you don't need to define a `glue__check_schema_exists` if you just want the `default__` macro used, just don't write anything! pretty nice feature if you ask me. saves us a lot of boilerplate